### PR TITLE
Added code for watermarking images

### PR DESCRIPTION
**Pull request by lsminter**
Added imports, JavaScript, and React Images for how to add watermarks to images.

Add imports and JavaScript.
Screenshot 2023-09-19 at 11 34 21 AM

React code to display images. 
Screenshot 2023-09-19 at 11 34 39 AM

### CODE REVIEW

File: examples/react-advanced-image/src/App.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
-import { Cloudinary } from '@cloudinary/url-gen';
+import { Cloudinary, Transformation } from '@cloudinary/url-gen';
 import { AdvancedImage, lazyload, placeholder } from '@cloudinary/react';
+import { fill, scale } from "@cloudinary/url-gen/actions/resize";
+import { source } from "@cloudinary/url-gen/actions/overlay";
+import { image, text } from "@cloudinary/url-gen/qualifiers/source";
+import { Position } from "@cloudinary/url-gen/qualifiers/position";
+import { compass } from "@cloudinary/url-gen/qualifiers/gravity";
+import { TextStyle } from "@cloudinary/url-gen/qualifiers/textStyle";
+import { opacity } from "@cloudinary/url-gen/actions/adjust";
```
Comment (lsminter) [inline examples/react-advanced-image/src/App.js:9]: Imports necessary to make the code work.

File: examples/react-advanced-image/src/App.js [hunk 2]
```diff
@@ -12,6 +19,37 @@ const cld = new Cloudinary({
...
+    .resize(fill(300, 300))
+    .format('auto')
+    .overlay(
+      source(
+        text('PREVIEW', new TextStyle('arial', 60)
+          .fontWeight('bold'))
+          .textColor('gray')
+          .transformation(new Transformation().adjust(opacity(70)))
+      )
+    )
```
Comment (lsminter) [inline examples/react-advanced-image/src/App.js:51]: My copied over JS.

File: examples/react-advanced-image/src/App.js [hunk 3]
```diff
@@ -70,14 +108,21 @@ function App() {
...
           })}
         </ul>
       </div>
 
+      <div className="container">
+        <h2>Watermarks</h2>
+        <p>Add watermarks to your images.</p>
+        <img src={myImage} />
+        <img src={mySecondImage} />
+      </div>
```
Comment (lsminter) [inline examples/react-advanced-image/src/App.js:124]: React code to display the images.

File: examples/react-advanced-image/src/App.js [hunk 4]
```diff
@@ -70,14 +108,21 @@ function App() {
                   height={image.height}
                   cldImg={cld.image(image.image).delivery('q_auto').format('auto')}
                   alt={image.title}
-                  plugins={[ lazyload(), placeholder() ]}
+                  plugins={[lazyload(), placeholder()]}
```
Comment (lsminter) [inline examples/react-advanced-image/src/App.js:111]: Prettier changed these to remove white space.

**Comment by colbyfayock** [comment]
thanks @lsminter - for future reference, id start off by cloning one of the existing blocks then adding transformations kinda like the other images, to a mapped list. but ill handle that for this one. 🚀

**Comment by colbyfayock** [comment]
@all-contributors please add @lsminter for code

**Pull request merged**